doc/syntax/syntax_en.tex

```diff
@@ -26,6 +26,8 @@
 
 \setlist[itemize]{noitemsep, topsep=0pt}
 
+\renewcommand{\MintedPygmentize}{../../_build/default/compiler/catala.exe pygmentize}
```

Comment:
Could you put somewhere in the README files or doc what `catala pygmentize` is for and how to use it for instance for powering LaTeX's `minted` ?

Reply:
Let's say I have some catala code in my latex and some c code somewhere else. Will `catala pygmentize` also support the c code?